tests/spread/extensions/env-injector/task.yaml

```
@@ -0,0 +1,84 @@
+summary: Build and run a basic hello-world snap using extensions
+
+systems:
+  - ubuntu-24.04
```

Spread is tricky to work with.  This isn't running in CI because you need a wildcard to match against `google:ubuntu-24.04-64`:

```suggestion
  - ubuntu-24.04*
```

---

Awesome! Thank you!